ckanext/data_qld_theme/templates/qa/openness_stars_brief.html

```diff
@@ -0,0 +1,30 @@
+{#
+Displays 5 starts of openness for a resource or dataset
```

Comment:
Misspelled "stars".

Reply:
Fixed spelling mistake.
FYI this spelling mistake was coped from original template https://github.com/ckan/ckanext-qa/blob/master/ckanext/qa/templates/qa/openness_stars_brief.html#L2